.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: Release
+
+on:
+  push:
+    branches: [master, next]
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PUBLIC_CDN_URL: 'https://embed.typeform.com'
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  init:
+    name: Init
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.ci-skip-step.outputs.ci-skip }}
+      skip-not: ${{ steps.ci-skip-step.outputs.ci-skip-not }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: mstachniuk/ci-skip@master
+        id: ci-skip-step
+        with:
+          commit-filter: '[skip ci]'
+  release_npm:
+    name: Release to NPM
+    runs-on: ubuntu-latest
+    needs: init
+    if: ${{ needs.init.outputs.skip == 'false' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          # Use custom token from repo secrets to allow semantic release to push commit:
+          # https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md#pushing-packagejson-changes-to-a-master-branch
+          persist-credentials: false
+          token: ${{ secrets.GH_TOKEN }}
+      - uses: actions/setup-node@v1
+      - uses: actions/cache@v2
+        with:
+          path: |
+            **/node_modules
+            ~/.cache
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - run: yarn install
+      - run: yarn build
+      # Update registry and tokens with write access for releasing.
+      - run: rm ./.npmrc
+      - run: npm config set '//registry.npmjs.org/:_authToken' $NPM_TOKEN
+      - run: npm config set '//npm.pkg.github.com/:_authToken' $GH_TOKEN
+      - run: npm config set @typeform:registry https://npm.pkg.github.com/
+      - run: yarn run lerna exec --concurrency 1 npx --no-install -- -- semantic-release
```

Comment:
`yarn lerna run semantic-release` does not work? Or do we need the concurrency  to be 1?

I would even suggest to run `yarn lerna run release` and implement `release` script in each package. For now it will be just `"release": "yarn semantic-release",` but later we can have customized release processes for each package.

Reply:
Maybe it would, I took it from here:
https://github.com/pmowrer/semantic-release-monorepo#with-lerna
I'm assuming having some concurrency might be interesting for building multiple packages